.circleci/config.yml

```diff
@@ -1,11 +1,35 @@
 version: 2
 jobs:
   bats-unit-test:
-    machine: true
+    docker:
+        # This image is built from test/docker/Test.dockerfile
+        - image: hashicorpdev/vault-helm-test:0.1.0
     steps:
       - checkout
-      - run: make test-image
-      - run: make test-unit
+      - run: bats ./test/unit -t
+  acceptance:
+    docker:
+        # This image is build from test/docker/Test.dockerfile
+        - image: hashicorpdev/vault-helm-test:0.1.0
+
+    steps:
+        - checkout
+        - run:
+            name: terraform init & apply
+            command: |
+                echo -e "${GOOGLE_APP_CREDS}" | base64 -d > vault-helm-test.json
+                export GOOGLE_CREDENTIALS=vault-helm-test.json
+                make provision-cluster
```

Comment:
The terraform code in this repo creates/deletes a KMS key.  The deletion of this key takes up to 24 hours.  Have you seen failures due to Terraform receiving an error from the Google API?

The KMS key is there for manual testing but isn't actually used by the acceptance tests.  This could be removed if it causes issues.

Reply:
Resolving, I now noticed you already removed it.